mount.yazi/main.lua

```diff
@@ -30,6 +30,14 @@ local update_cursor = ya.sync(function(self, cursor)
 	ya.render()
 end)
 
+local function get_fstype(p)
+	local cmd = "lsblk -n -o fstype" .. " " .. p
+	local hnd = io.popen(cmd)
```

Comment:
Please use [`Command`](https://yazi-rs.github.io/docs/plugins/utils/#command) instead of `io.popen()`